book_class.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    Rental.new(date, self, person)
+  end
```

Comment:
- [] Kindly make sure that when passing in the parameter to the `Rental` class, they are passed in the right order according to the constructor. Currently, the Rental need the following parameters in the following order to be created:
https://github.com/lucabenaventew/school-library/blob/57b5ef3bdc32e1f2c2356a9c49cd83da49d215c3/rental_class.rb#L4